opentracing/README.md

```diff
@@ -8,37 +8,33 @@ For background:
 
 Everyday consumers of this `opentracing` package really only need to worry
 about a couple of key abstractions: the `StartSpan` function, the `Span`
-interface, and golang's `context.Context` idiom. Here are code snippets
+interface, and binding a `Recorder` at `main()`-time. Here are code snippets
 demonstrating some important use cases.
 
-A note about `context.Context`: the opentracing API here encourages (but does
-not require) the use of golang-team's `context.Context` scheme. Whether we like
-it or not (this author is on the fence), `context.Context` is here to stay,
-goroutine-local-storage bedamned. When in Rome, do as the Romans do (etc).
-
 #### Singleton initialization
 
-The simplest starting point is `./global.go`. As early as possible, call
+The simplest starting point is `./default_tracer.go`. As early as possible, call
 
     import ".../opentracing"
     
     func main() {
-        tracingRecorder := ... // tracing impl specific
-        tracingContextIDSource := ... // tracing impl specific
-        opentracing.InitGlobalTracer(tracingRecorder, tracingContextIDSource)
+        procRecorder := some_tracing_impl.NewRecorder(...) // tracing impl specific
+        traceContextSource := some_tracing_impl.NewContextSource(...) // tracing impl specific
+        opentracing.InitDefaultTracer(procRecorder, traceContextSource)
         ...
     }
 
-##### Note: the singletons are optional
+##### Non-Singleton initialization
 
-If global singletons make you sad, use `opentracing.NewStandardTracer(...)`
-directly and manage ownership of the `opentracing.OpenTracer` explicitly.
+If you prefer direct control to singletons, use
+`opentracing.NewStandardTracer(...)` directly and manage ownership of the
+`opentracing.OpenTracer` implementation explicitly.
 
-#### Creating a Span given an existing Golang `context.Context`
+#### Creating a root Span (i.e., without a known parent)
```

Comment:
May be: `Starting a brand new trace by creating a root span (i.e., without a known parent)`